src/components/phaseAccordions/PhaseAccordions.tsx

```diff
@@ -0,0 +1,74 @@
+import moment from 'moment'
+import { Accordion, ContentBlock, TaskContentIcon } from '@/components/index'
+import type { AccordionState } from '@/hook/useAccordion'
+import type { Status, TaskDTO, TaskContent } from '@/data/phase/dto.type'
+
+type PhaseAccordionProps = Readonly<{
+  activeAccordion: AccordionState
+  onClick: (taskName: string) => () => void
+  number: number
+  phaseName: string
+  status: Status
+  tasks: TaskDTO[]
+}>
+
+export const formatString = (text: string): string => text.trim().toLowerCase()
+
+const PhaseAccordions: React.FC<PhaseAccordionProps> = ({
+  activeAccordion,
+  phaseName,
+  status,
+  tasks,
+  onClick,
+  number
+}): JSX.Element => (
+  <div className="okp4-nemeton-web-page-accordions-wrapper">
+    <div className="okp4-nemeton-web-page-accordions-title-container">
+      <h3>Phase {number}</h3>
+      <h2 id={phaseName.toLowerCase()}>
+        {phaseName}
+        {status === 'closed' && <span> (closed)</span>}
+      </h2>
+    </div>
+    {tasks.map(({ taskName, taskContent, taskDuration }, index) => {
+      const { from, to } = taskDuration
+      const title = (
+        <div className="okp4-nemeton-web-tasks-accordion-title">
+          <h3>{taskName}</h3>
+          <p>{`${moment(from).utc().format('MMM. Do, H:mm ')} UTC - ${moment(to)
+            .utc()
+            .format('MMM. Do, H:mm ')} UTC`}</p>
+        </div>
+      )
+      const accordionId = formatString(`${phaseName}-${taskName}`)
+      const active = activeAccordion === accordionId
+
+      return (
+        <div key={index}>
+          <Accordion
+            content={
+              <>
+                {taskContent.map(
+                  ({ id, title, contentDescription }: TaskContent): JSX.Element => (
+                    <div key={id}>
+                      <ContentBlock
+                        description={contentDescription}
+                        icon={<TaskContentIcon id={id} />}
+                        title={title}
+                      />
+                    </div>
+                  )
+                )}
+              </>
+            }
+            isExpanded={active}
+            onToggle={onClick(accordionId)}
```

Comment:
You pass the `accordionId`value, which is the concatenation of the `phaseName` and the `taskName`, however, AFAIS, the definition of the `onClick` expects a `taskName` instead.

Reply:
@houssemmimoun27 have you handled this comment?

Reply:
@fredericvilcot 
This comment is fixed.

Reply:
@houssemmimoun27 no, it's not. `accordionId` is the concatenation of the `phaseName` and the `taskName`, but the `onClick` function the you call says that it should be a `taskName` according to its declaration: `onClick: (taskName: string) => () => void`. It's a question of consistency when it comes to knowing what is expected of the caller, and what we get back from the called.

Reply:
@ccamel 
You're right. But I changed the definition of the `onClick` to `accordionId` instead of `taskName`.

```ts
 const handleClick = useCallback(
 (accordionId: string) => () => setActiveTask(activeTask === accordionId ? null : accordionId),
 [activeTask, setActiveTask])
```

Reply:
From what I see here:

```
type PhaseAccordionProps = Readonly<{
  activeAccordion: AccordionState
  onClick: (taskName: string) => () => void
  number: number
  phaseName: string
  status: Status
  tasks: TaskDTO[]
}>
```

If I'm not wrong the `onClick` function is expected to accept a `taskName`, which is not the case since you call it with the `accordionId` value. So, I would like you to make the change to make it consistent, because, as a reminder, props serve as an external interface through which we gain knowledge about how the component can be used. It is also important that the naming is clean.